### PR TITLE
Adds evolving web tooling blog post

### LINKED ISSUE #5460: Evolving Cumulative Layout Shift in Web Tooling [6/1/2021]
[Draft](https://docs.google.com/document/d/1raK5Pq2yuDmoDujIL23AP18e83XJjQALDjIJbHcnpLw/edit?ts=60afbc09#heading=h.dse537eswc1m)

### PR DESCRIPTION
**Pull request by rachelandrew**
Fixes #5460 Evolving CLS in Web Tooling.

### CODE REVIEW

**Comment by rachelandrew** [comment]
@mihajlija @addyosmani @egsweeny here is the PR, so any changes made in the last 30 minutes or so you'll need to make here!

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 1]
```diff
@@ -0,0 +1,270 @@
+---
+title: Evolving Cumulative Layout Shift in Web Tooling
```
Comment (mihajlija) [inline src/site/content/en/blog/cls-web-tooling/index.md:2]:
```suggestion
title: Evolving Cumulative Layout Shift in web tooling
```

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+  Lighthouse, PageSpeed Insights, and Chrome UX Report.
+date: 2021-06-02
+hero: image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/UAslFmRMON2y5qtY1TZE.png
+alt: The Lighthouse Scoring Calculator
+description: >
+  Starting today,
+  a change to CLS has been rolled out across a number of Chrome's web tooling surfaces including
+  Lighthouse, PageSpeed Insights, and Chrome UX Report.
+tags:
+  - blog
```
Comment (mihajlija) [inline src/site/content/en/blog/cls-web-tooling/index.md:15]:
```suggestion
  - blog
  - web-vitals
```

**Review by mihajlija — APPROVED** [review]

File: src/site/content/en/blog/cls-web-tooling/index.md (hunk 1 above)
Comment (mihajlija) [reply src/site/content/en/blog/cls-web-tooling/index.md:2]: @rachelandrew Please check that titles and headings use sentence case: https://web.dev/handbook/grammar/#titles-and-headings

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+    }
+  }).observe({type: 'layout-shift', buffered: true});
+}
+```
+
+Or by relying directly on the
+[Web Vitals JavaScript library](https://github.com/GoogleChrome/web-vitals),
+which has also been updated with this change.
+
+## Additional Updates
```
Comment (mihajlija) [inline src/site/content/en/blog/cls-web-tooling/index.md:246]:
```suggestion
## Additional updates
```

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 4]
```diff
@@ -0,0 +1,270 @@
+---
+title: Evolving Cumulative Layout Shift in Web Tooling
+subhead: >
+  Starting today,
+  a change to CLS has been rolled out across a number of Chrome's web tooling surfaces including
+  Lighthouse, PageSpeed Insights, and Chrome UX Report.
+date: 2021-06-02
```
Comment (mihajlija) [inline src/site/content/en/blog/cls-web-tooling/index.md:7]:
```suggestion
date: 2021-06-01
```

Comment (mihajlija) [reply src/site/content/en/blog/cls-web-tooling/index.md:7]: Hoping this comes through today :)

**Comment by mihajlija** [comment]
@addyosmani @egsweeny Preemptively approved so you can merge the post when ready.

**Comment by addyosmani** [comment]
Thank you, @mihajlija and @rachelandrew! We'll highlight any additional changes before this goes live.

**Comment by addyosmani** [comment]
As an update, we're going to delay the post going out until early tomorrow morning. There are some late-breaking pipeline changes to PSI and CrUX which we'll be rolling out tonight (still within the June 1st period per post) but otherwise content of the post should remain same.

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+
+You will be able to continue relying on this data to be present for a period of up to 6 months,
+with uncapped CLS being retired on December 14th, 2021.
+
+## Updating the CLS weighting in Lighthouse
+
+When CLS was first introduced in Lighthouse,
+it was a brand new sparkly metric.
+As such, in order to make sure developers had time to test, benchmark,
+and optimize against it,in the performance score CLS was weighed less heavily.
```
Comment (philipwalton) [inline src/site/content/en/blog/cls-web-tooling/index.md:200]: This line is awkward (also there's a missing space after the comma). Can we reword to this?

```suggestion
and optimize against it, CLS was weighed less heavily in the performance score.
```

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+
+You can find the updated weightings of metrics in Lighthouse v8 in the
+[scoring calculator](https://googlechrome.github.io/lighthouse/scorecalc/).
+
+{% Img
+src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/UAslFmRMON2y5qtY1TZE.png",
+alt="Lighthouse scoring calculator",
+width="800", height="405" %}
+
+Lighthouse 8.0's CLS implementation includes both windowing and CLS contribution from subframes. Prior to 8.0, CLS in Lighthouse didn't include subframes' CLS in the metric calculation, but now does. And just for confirmation, field CLS measured by CrUX handles also handles windowing and subframes similarly.
```
Comment (philipwalton) [inline src/site/content/en/blog/cls-web-tooling/index.md:215]:
```suggestion
Lighthouse 8.0's CLS implementation includes both windowing and CLS contribution from subframes. Prior to 8.0, CLS in Lighthouse didn't include subframes' CLS in the metric calculation, but now does. And just for confirmation, field CLS measured by CrUX also handles windowing and subframes similarly. 
```

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+Lighthouse 8.0's CLS implementation includes both windowing and CLS contribution from subframes. Prior to 8.0, CLS in Lighthouse didn't include subframes' CLS in the metric calculation, but now does. And just for confirmation, field CLS measured by CrUX handles also handles windowing and subframes similarly. 
+
+Still, the primary difference between lab and field CLS is that lab CLS's window of observation ends at "fully loaded" as determined under lab conditions, whereas in the field, the window of observation extends to the entire page lifetime, including post-load activity. That said, the windowing adjustment does mitigate this difference substantially.
+
+## Measuring in the field yourself
+
+Should you wish to measure the latest CLS implementation,
+you can also record this for your field data via RUM using the following PerformanceObserver snippet:
+
+```js
```
Comment (philipwalton) [inline src/site/content/en/blog/cls-web-tooling/index.md:224]: Can we link to this page instead, so we don't have two different code snippets in two different posts?
https://web.dev/cls/#measure-cls-in-javascript

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 8]
```diff
@@ -0,0 +1,278 @@
...
+```
+
+Or by relying directly on the
+[Web Vitals JavaScript library](https://github.com/GoogleChrome/web-vitals),
+which has also been updated with this change.
+
+## Additional updates
+
+Outside of the updates to Cumulative Layout Shift,
+the following metrics related updates have also been made to our web tooling:
```
Comment (philipwalton) [inline src/site/content/en/blog/cls-web-tooling/index.md:254]:
```suggestion
the following metrics-related updates have also been made to our web tooling:
```

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 9]
```diff
@@ -0,0 +1,278 @@
...
+    }
+  }).observe({type: 'layout-shift', buffered: true});
+}
+```
+
+Or by relying directly on the
+[Web Vitals JavaScript library](https://github.com/GoogleChrome/web-vitals),
+which has also been updated with this change.
+
+## Additional updates
```
Comment (philipwalton) [inline src/site/content/en/blog/cls-web-tooling/index.md:251]: Super nit: it's fine to include these here if we don't have a better place to post them at the moment, but it feels a bit weird to mention LCP and FCP updates in a post about CLS.

File: src/site/content/en/blog/cls-web-tooling/index.md [hunk 10]
```diff
@@ -0,0 +1,278 @@
...
+
+Outside of the updates to Cumulative Layout Shift,
+the following metrics related updates have also been made to our web tooling:
+
+- We're updating to
+[the most recent definition of the Largest Contentful Paint metric](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/2020_11_lcp.md).
+CrUX API, PSI, PSI API, Search Console will update on June 1, 2021. CrUX BigQuery will update on June 8, 2021.
+- In CrUX, First Contentful Paint tri-binning thresholds have been updated to be,
+Good: [0-1.8s], Needs Improvement: (1.8s-3s), Poor: [3s-∞]
```
Comment (philipwalton) [inline src/site/content/en/blog/cls-web-tooling/index.md:260]: Shouldn't threshold bucket upper bounds always  be inclusive? (cc: @rviscomi)

```suggestion
Good: [0-1.8s], Needs Improvement: (1.8s-3s], Poor: (3s-∞]
```

Comment (rviscomi) [reply src/site/content/en/blog/cls-web-tooling/index.md:260]: +1 but it's complicated for CrUX BigQuery histograms which are exclusive of the upper bound 😕

File: src/site/content/en/blog/cls-web-tooling/index.md (hunk 9 above)
Comment (rviscomi) [reply src/site/content/en/blog/cls-web-tooling/index.md:251]: I think this was my suggestion, for lack of a better place. Agreed it's not ideal but maybe the best we can do.

File: src/site/content/en/blog/cls-web-tooling/index.md (hunk 7 above)
Comment (addyosmani) [reply src/site/content/en/blog/cls-web-tooling/index.md:224]: Ack. I'll be updating this in a subsequent commit shortly.

**Pull request merged**